scripts/uninstall.ts

```diff
@@ -1,11 +1,13 @@
 import { GoogleAnalytics } from "../lib/GoogleAnalytics";
 
 function uninstall() {
-	GoogleAnalytics.post({
-		t: "screenview",
-		// tslint:disable-next-line:object-literal-sort-keys
-		cd: `uninstall global: ${!!process.env.npm_config_global}`
-	});
+	if (!!process.env.npm_config_global) {
+		GoogleAnalytics.post({
+			t: "screenview",
+			// tslint:disable-next-line:object-literal-sort-keys
```

Comment:
Why don't you just sort the two keys, instead of disabling the line?

Reply:
We are listing post() parameters everywhere in same order:

1. `t` - type of post;
2. `cd` - screen name, or `ec` - event category;
3. `el` - event label, if applicable;
4. `ea` -event action, if applicable;
5. `cd[1..12]` - custom dimensions if any.

This is why we have these two in this order.

Reply:
Finally we decided to change tslint config and added:

`"object-literal-sort-keys": [ true, "ignore-case", "match-declaration-order", "shorthand-first" ]`

This will force lint to check object's keys order against its interface if any. This allows us to remove 

`// tslint:disable:object-literal-sort-keys`

line from most of the code.